README.md

```diff
@@ -255,6 +257,12 @@ File extensions are still relevant. While either a `.js` or an `.mjs` file can b
 
 The CommonJS automatic file extension resolution or folder `index.js` discovery are not supported for `import` statements, even when referencing files inside CommonJS packages. Both `import cookies from 'request/lib/cookies'` and `import request from './node_modules/request'` would throw. Automatic file extension resolution or folder `index.js` discovery _are_ still supported for `package.json` `"main"` field specifiers, however, to preserve backward compatibility.
 
+#### Initial entry point
+
+It is outside the scope of this proposal to define how an ESM-supporting Node should determine the parse goal, ESM or CommonJS, for every type of input (file, string via `--eval`, string via `STDIN`, extensionless file, etc.). However, for the purposes of completeness for this proposal we will define _one_ method that we expect Node to support, with the understanding that additional methods will be necessary to handle the other use cases.
+  
+To preserve backward compatibility, we expect that `node file.js` will continue to load the entry point `file.js` as CommonJS by default. (This may be deprecated and eventually changed to an ESM default in the future, but certainly not in the initial release of Node with ESM support.) However, if `file.js` is inside an ESM package scope, Node should load `file.js` as ESM. So for example, if a `package.json` with an `"exports"` key is in the same folder as `file.js`, `node file.js` would load `file.js` as ESM. The same “search up the file tree for a `package.json`” algorithm for `import` statements applies when determining the package scope for `file.js`.
```

Comment:
Instead of mentioning `"exports"` key by name you could make the section that describes `"A package.json file is detected as ESM"` as a section and link to it as `"is [inside an ESM package scope](#the-section-anchor)"`.

Reply:
Since that was just adding a heading, I made that change directly without making a PR and then rebased this on top of it.

Reply:
Cool.